web-react-ts/src/components/RegisterUserCard.tsx

```diff
@@ -29,36 +36,31 @@ const RegisterUserCard = ({ user }: { user: UserData }) => {
     currentUser?.email as string
   )
 
-  const { data: adminDoc } = useFirestoreDocData(adminDocReference)
+  const { data: adminDoc, status } = useFirestoreDocData(adminDocReference)
```

Comment:
no type for status??

Reply:
The adminDoc isn't a type, it's like a destructured object assignment.